src/main/java/ch/epfl/biop/wrappers/cellpose/CellposeTaskSettings.java

```diff
@@ -59,23 +58,53 @@ public CellposeTaskSettings setFlowTh(double flow_threshold) {
         return this;
     }
 
-    /*
+    public CellposeTaskSettings setCellProbTh(double cellprob_threshold) {
+        this.cellprob_threshold = cellprob_threshold;
+        return this;
+    }
+
     public CellposeTaskSettings setAnisotropy(double anisotropy) {
         this.anisotropy = anisotropy;
         return this;
     }
-    */
-    public CellposeTaskSettings setCellProbTh(double cellprob_threshold) {
-        this.cellprob_threshold = cellprob_threshold;
+
+    public CellposeTaskSettings setStitchThreshold(double stitch_threshold){
+        this.stitch_threshold = stitch_threshold;
         return this;
     }
 
+    public CellposeTaskSettings setDo3D() {
```

Comment:
I would remove this one if you have already do3D (boolean)

Reply:
was the original one, but now it also needs a setter with input true/false